user_guide/content/modules/working_with_an_existing_work_item.adoc

```diff
@@ -1,90 +1,68 @@
 [#working_with_an_existing_work_item]
 = Working with an Existing Work Item
-//redo this, stopped here
-Use the Planner in the *Plan* tab to track your Work Items and Iterations using either the list view or the board view. The *Plan* tab is split into two panes:
 
-* The left pane displays the *Backlog* and *Board* views of Iterations and Work Items. For more information see <<about_iterations,About Iterations>>.
-* The right pane displays the Work Items within the selected Iteration or the backlog.
+Click btn:[Plan] at the top of the page to view the Planner.
 
-You can plan and track your Work Items in the right pane using either the hierarchical or flat list view of Work Items on the menu:Plan[Backlog] subtab or the Kanban-based board view on the menu:Plan[Board] subtab.
+The *Backlog* view is the default view in the Planner and it displays all the Iterations included in the space. Iterations are sorted into one of the following categories:
 
```

Comment:
The information on line 9 is the crucial difference between the list view and board view. The Plan tab is split into two Panes,(line 4) one talks about Iterations the other about Work Items. 
Work Items are displayed in two ways, List and Kanban style. In the Backlog view, Work items are displayed in a list view, this part should be mentioned in the introductory paragraph, imho. User has the choice to work with Work items either using the list view or the Board view based on his choice.
Currently, in the Backlog section, Iterations are being described (line 8) which is mentioned in detail in the Iterations topic. We can simply link to the section as in line 6. Can we discuss this please?